### DISCUSSION TITLE
General query regarding things to carry in US

### DISCUSSION THREAD

**Discussion by LovleenKaur-tal** [snigi-gupta/AttendingUniversityAtBuffalo#54] [General]
Hi Snigdha!

I have few doubts related to : -

1. Can we carry Sewing Kits?
2. What all medicines should be carries while travelling to US and how much should be the permissible quantity of each medicine?
3. Is it judicious to bring Beddings(Bedsheets, Blanket etc) from India? Or shall we buy from US? Which one is better/ cost effective?
4. Can we bring Dal/Wheat/Rice from India? Is it allowed?

**Comment by snigi-gupta** [comment]
@LovleenKaur-tal 

1. yes (in check-in)
2. all meds you usually use. get prescription for those from local doc. antibiotics, paracetamol, cough drop/syrup etc can be carried in substantial quantities.
3. NO (answered in discussions)
4. NO (nothing WHITE allowed) DONT bring all this and waste your space. Bring useful things.
